facebook/OpenBIC
# wc: mb: Add source identify function and project command

Summary:
- Add source identification between main source and second source.
- Add project command.

Test Plan:
- Build Code: PASS
- Check project info command in BIC console:

Log:
- BIC console:
  ```
  uart:~$ wcmb info
  ========================{PLATFORM INFO}========================
  * NAME:          Waimea Canyon platform command
  * DESCRIPTION:   Commands that could be used to debug or validate.
  * Note:          none
  ------------------------------------------------------------------
  * SYSTEM CLASS:  Dual
  * SYSTEM SOURCE: Sencond
  ========================{PLATFORM INFO}========================
  ```
